### DISCUSSION TITLE
How to set sphinx default-domain?

### DISCUSSION THREAD

**Discussion by hyanwong** [executablebooks/meta#399] [Q&A] [answered]
I'm reposting this from StackOverflow, as no-one there seems to have a suggestion. I want to specify [.. default-domain::mypackage](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-default-domain) on a single jupyterbook page, so that I can refer to ``{func}`.myfunc` `` rather than ``{func}`mypackage.myfunc` `` in the markdown file. I have several packages referenced via intersphinx, and I only want to use `mypackage` as the default on this page, not on other pages of the jupyterbook. Is there a way of doing this, and what's the correct syntax?

**Comment by hyanwong** [comment, marked as answer]
Ha, just found it. It seems that this will do the trick:

    ```{currentmodule} mypackage
    ```

**Reply by chrisjsewell** [reply]
Always good, when someone answers their own question, thanks 😄. I have marked this as the answer yes?

**Reply by hyanwong** [reply]
Thanks. I'm slightly unclear on the distinction between `default-domain` and `currentmodule`, but the latter seems to work in my case.
